app/entities/user.go

```diff
@@ -15,6 +15,14 @@ type User struct {
 	CreatedAt string `json:"created_at"`
 }
 
+type PublicUser struct {
```

Comment:
Wrong package. Better not `PublicUser` but `UserResponse` or `User`

Reply:
Had moved `PublicUser` to rest package with name `User`